Modules/Loadable/SubjectHierarchy/Widgets/qSlicerSubjectHierarchyPluginLogic.h

```diff
@@ -71,9 +71,17 @@ class Q_SLICER_MODULE_SUBJECTHIERARCHY_WIDGETS_EXPORT qSlicerSubjectHierarchyPlu
 
   /// Register subject hierarchy core plugins
   /// Note: Registering plugins provided by other modules is the responsibility
-  ///   of the module!
+  ///       of the module!
   void registerCorePlugins();
 
+  /// Get all view menu actions available
+  /// \return List of object names of the view menu actions
+  Q_INVOKABLE QStringList allViewMenuActions();
+  /// Set desired set of view menu actions
+  /// \param actionObjectNames List of view menu actions to consider. Only actions included here by object name
+  ///        are shown in the menu if they are accepted by the owner plugin. The order set here is used.
+  Q_INVOKABLE void setViewMenuActionWhitelist(QStringList actionObjectNames);
```

Comment:
I've checked naming conventions for similar methods in Qt and found that "whitelist" is only used once. I think it would be better have these method names instead:
- setVisibleViewMenuActions or setViewMenuActions (rename from setViewMenuActionWhiteList)
- visibleViewMenuActions or viewMenuActions (new)
- allViewMenuActions or allAvailableViewMenuActions (no change needed)

Reply:
`setViewMenuActions()` / `viewMenuActions()`  :+1: